com.gitlab.fossbr.WhatsAppQT.yml

```diff
@@ -0,0 +1,41 @@
+app-id: com.gitlab.fossbr.WhatsAppQT
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.15-21.08'
+command: WhatsAppQT
+finish-args:
+- "--share=ipc"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--socket=pulseaudio"
+- "--share=network"
+- "--filesystem=xdg-download"
+- "--device=dri"
+- "--env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess"
+- "--talk-name=org.kde.StatusNotifierWatcher"
+- "--own-name=org.kde.*"
+cleanup-commands:
+- /app/cleanup-BaseApp.sh
+modules:
+- name: WhatsAppQT
+  buildsystem: qmake
+  build-commands:
+    - install -Dm00644 ./app/share/metainfo/$FLATPAK_ID.appdata.xml $FLATPAK_DEST/share/metainfo/$FLATPAK_ID.appdata.xml
+    - install -Dm00644 ./app/share/applications/$FLATPAK_ID.desktop $FLATPAK_DEST/share/applications/$FLATPAK_ID.desktop
```

Comment:
if you want to rename them use `rename-desktop-file` and `rename-appdata-file` instead.

also reading this that means you don't need the appdata/metainfo file you have added in this repository, since it's not even installed anyway.